### PR TITLE
Add tests for foils and take foil generation out of experiment.js

### PR DESCRIPTION
**Pull request by sarah-witt**

### CODE REVIEW

File: test/foils.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    );
+  });
+
+  it("is not the same as exemplars", () => {
+    assert.notDeepEqual(
+      exemplars,
+      foils
+    );
+  });
+  it("Creates unique foils", () => {
```
Comment (ryandrew14) [inline test/foils.test.js]: I'm being annoying here but could you make the descriptions all begin with lowercase characters? Just to keep the formatting consistent

**Review by ryandrew14 — CHANGES_REQUESTED** [review]
This is awesome and I'm really excited about it. One tiny comment. Thanks!!!

File: test/foils.test.js (hunk 1 above)
Comment (sarah-witt) [reply test/foils.test.js]: Got it! I also cleaned up more of the wording to make more sense

**Review by ryandrew14 — APPROVED** [review]
Looks great 🚀

**Pull request merged**